lib/src/dartdoc.dart

```diff
@@ -204,9 +204,6 @@ class Dartdoc {
     var libs = packageGraph.libraryCount;
     logInfo("Initialized dartdoc with $libs librar${libs == 1 ? 'y' : 'ies'}");
 
-    // Create the out directory.
-    if (!_outputDir.exists) _outputDir.create();
```

Comment:
This is created later on when actually generating docs (or at least, not creating it doesn't seem to prevent doc generation).